src/unpatched-fonts/CascadiaCode/LICENSE

```diff
@@ -0,0 +1,94 @@
+Copyright (c) 2019 - Present, Microsoft Corporation,
+with Reserved Font Name Cascadia Code.
+
```

Comment:
src/unpatched-fonts/CascadiaCode/Cascadia.ttf

Reply:
@GhosT02101991 what does this mean?

Reply:
I believe that the font can be changed, but any modified font could not use the name Cascadia Code, as looks like it is MS Trademark

Reply:
Right. I guess I was confused why it was being pointed out. Yeah the name needs to be respected due to RFN